jvm/src/test/scala/com/avast/sst/jvm/system/console/ConsoleModuleTest.scala

```diff
@@ -3,9 +3,9 @@ package com.avast.sst.jvm.system.console
 import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
 
 import cats.effect.SyncIO
+import org.scalatest.funsuite.AnyFunSuite
```

Comment:
I'm not sure I like that scalatest imports are before `scala.*`. I kind of prefer test related imports to be last.

Reply:
I'm afraid `SortImports` can't recognize which imports are test libraries and which are not :man_shrugging: